src/test/kotlin/io/github/rybalkinsd/kohttp/Utils.kt

```diff
@@ -11,3 +16,22 @@ fun assertResponses(expected: Map<String, String>, actual: JsonNode) {
         assertEquals(u, actual[t]?.asText())
     }
 }
+
+fun getSuccessfulResponsesAmount(
+        results: List<Deferred<Response>>
+): Int = runBlocking {
+    results.map { getCode(it) }
+            .filter { it == 200 }
+            .size
+}
+
+suspend fun getCode(
+        result: Deferred<Response>
```

Comment:
what is the reason for such line breaks?

Reply:
My fault, Idea was configured with such breaks by default